src/App.tsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable */
```

Comment:
The use of `/* eslint-disable */` is generally discouraged as it disables all ESLint rules for the entire file. Consider using more specific rule disabling or addressing the linting issues directly.